tests/test_raster.py

```diff
@@ -1,7 +1,11 @@
 import numpy as np
 import pytest
 
-from asf_tools import raster
+from osgeo import gdal
+
+from asf_tools import util, raster
+
+gdal.UseExceptions()
```

Comment:
```suggestion
from asf_tools import raster
```

Reply:
does this pull `gdal.UseExceptions()` out, don't we want that

Reply:
If we're not invoking anything from the `gdal` module, there's no need to tell it to raise exceptions